### ISSUE TITLE
[bug?] Do not update id.dat with spoofed FW version

### ISSUE THREAD

**Issue by windsurfer1122** [henkaku/henkaku#103]
When using HENkaku with Ensō then id.dat gets updated with the spoofed FW version.
This leads to an unaccessible memory card when deinstalling Ensō. Just happened to me (*).

First I wanted to report this as an Ensō bug, but regarding https://github.com/henkaku/enso/issues/11 it should be reported as a HENkaku bug.

Please consider changing the behaviour of updating the id.dat. Or re-open the Ensō bug as valid and create a workaround in the Ensō deinstallation code.
Thanks a lot in advance.

Fat Vita on FW 3.65 plus h-encore plus Ensō 1.1 installer.

P.S.:
(*) By accident I forgot to unlink my memory card (as I'm aware of this issue) and got trapped in a "You have to upgrade" boot.
Fortunately I could directly fix this by unlinking the mc in another Ensō enabled console.
Another solution would have been to temporally use another spare memory card (if you have one), format it and install Ensō again to unlink the original mc.

**Comment by yifanlu** [comment]
Yeah this is a known issue and I thought I fixed it a while ago but I’ve still seen it pop up time and time again and it’s pretty annoying. I already patched one point that updates the is.dat but there’s likely other points I missed.
